Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/pages/OptionsMenu.java

```diff
@@ -0,0 +1,66 @@
+package com.automattic.simplenote.smoke.pages;
+
+import com.automattic.simplenote.R;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+public class OptionsMenu extends BasePage {
+
+    private static final Integer POSITION_MENU = 2;
```

Comment:
What is this for? I don't see it used anywhere but I see some references to locating menu items by position — maybe leftover from an unused approach you tried earlier?

Reply:
Yes, as you said, it is totally leftover part from a changed approach. It will be removed.